apps/ui/src/components/inputs/text.tsx

```diff
@@ -1,110 +1,64 @@
-import {
-  Box,
-  Input as ChakraInput,
-  InputGroup,
-  InputLeftAddon,
-  InputRightAddon,
-  type InputProps,
-} from '@chakra-ui/react';
-import type { ReactNode } from 'react';
+import { Box, Flex, InputGroup, type BoxProps } from '@chakra-ui/react';
+import { useScreenSize } from '../../hooks/useScreenSize';
 
-interface CustomInputProps extends InputProps {
-  inputHeight?: number;
-  inputColor?: string;
-  value?: string;
-  leftAddon?: boolean;
-  leftAddonName?: string | ReactNode;
-  leftAddonWidth?: string | string[];
-  leftAddonColor?: string;
-  rightAddon?: boolean;
-  rightAddonName?: string | ReactNode;
-  rightAddonClick?: () => void;
-  rightAddonWidth?: string | string[];
-  rightAddonColor?: string;
+interface TextValueProps extends BoxProps {
+  content?: string;
+  leftAction?: string | React.ReactNode;
+  rightAction?: string | React.ReactNode;
 }
 
-const TextInput = (props: CustomInputProps) => {
-  const {
-    value,
-    inputHeight,
-    inputColor,
-    leftAddon,
-    leftAddonName,
-    leftAddonWidth,
-    leftAddonColor,
-    rightAddon,
-    rightAddonName,
-    rightAddonColor,
-    rightAddonWidth,
-    rightAddonClick,
-    ...rest
-  } = props;
+const TextValue = ({
+  content,
+  leftAction,
+  rightAction,
+  ...rest
+}: TextValueProps) => {
+  const { isMobile } = useScreenSize();
+
   return (
-    <Box w="full" display="flex" alignItems="center">
-      <InputGroup>
-        {leftAddon && (
-          <InputLeftAddon
-            h={inputHeight ?? 10}
-            bgColor="input.600"
-            borderColor="stroke.500"
-            borderRight="none"
-            alignItems="center"
-            justifyContent="flex-start"
-            fontSize="sm"
-            w={leftAddonWidth ?? 'min-content'}
-            color={leftAddonColor ?? 'section.500'}
-            borderLeftRadius="xl"
-          >
-            {leftAddonName}
-          </InputLeftAddon>
-        )}
-        <ChakraInput
-          defaultValue={value ?? ''}
-          type="text"
-          readOnly={true}
-          h={inputHeight ?? 10}
+    <Flex
+      w="full"
+      gap={4}
+      justifyContent="space-between"
+      alignItems="center"
+      // Add all styles same used in input
+    >
+      <InputGroup w="full">
```

Comment:
Only to be able to use the Left and Right addon if needed. The style without it would be tha same, just make sure to style your component to match with the middle box.